src/components/NcChip/NcChip.vue

```diff
@@ -171,24 +171,24 @@ const slots = defineSlots<{
 	/**
 	 * The actions slot can be used to add custom actions (`NcAction*`) to the chips actions.
 	 */
-	actions?: Slot
+	actions: Slot
```

Comment:
```suggestion
	actions?: Slot
```
`slots.actions` is not guaranteed to be a function. When a slot is not passed, `slots.name` is `undefined`.

The previous type was correct.

Same as in https://github.com/nextcloud-libraries/nextcloud-vue/pull/6812/